eng/versioning/external_dependencies.txt

```diff
@@ -98,6 +98,10 @@ org.openjdk.jmh:jmh-generator-annprocess;1.22
 org.spockframework:spock-core;1.3-groovy-2.5
 org.testng:testng;6.14.3
 uk.org.lidalia:slf4j-test;1.2.0
+org.unitils:unitils-core;3.4.2
+com.microsoft.azure:azure;1.24.1
+com.microsoft.azure:azure-mgmt-search;1.24.1
+com.azure:azure-storage-blob;12.1.0
```

Comment:
+@alzimmermsft @srnagar 
You're building your new libraries in the client sdk track and com.azure:azure-storage-blob is one of the libraries that's also built as part of the client sdk. The current released version of [com.azure:azure-storage-blob](https://search.maven.org/artifact/com.azure/azure-storage-blob) is 12.2.0-beta.1 which was released on December 17th. Any dependencies on com.azure:azure-storage-blob inside of the client track should be using the dependency version inside of version_client.txt. External dependencies are for things we don't built or things built from a different track.

Reply:
@JimSuplizio what would be the way to use older versions than the current ones? ie azure-storage-blob;12.1.0?

Reply:
> @JimSuplizio what would be the way to use older versions than the current ones? ie azure-storage-blob;12.1.0?

@navalev dependencies should be targeting the latest released version of a library whenever possible. If there's a compelling reason not to then it would be something we'd have to discuss.

Reply:
@JimSuplizio  I swithched the azure-storage-blob version to 12.2.0-beta.1 and ever since than the build fails on "Test Windows from source - java 8" task (all other tasks pass). Any suggestion on how to resolve this?

Reply:
@navalev please take a look at your pom.service.xml file for search, where you've got the client dependencies, including test dependencies, listed there but you're missing the entry for azure-storage-blob so it isn't building it like it should be.

Reply:
@JimSuplizio thanks! that sorted this out.